src/controllers/controller-types.ts

```diff
@@ -84,3 +84,10 @@ export interface UserMessage {
   success?: string;
   fail?: string;
 }
+
+export interface UserOrgPage extends UserObj {
+  orgRolesCount: number;
+  spaceRolesCount: number;
+  daysToExpiration: number | null;
+  lastLogonTime: number | null;
```

Comment:
[non-blocking question] Does it matter that in the controller, you potentially assign `undefined` to `lastLogonTime` rather than null?

Reply:
Yep good catch, I'll add `undefined` as an option.

Reply:
done